packages/block-library/src/image/index.php

```diff
@@ -30,14 +31,31 @@ function render_block_core_image( $attributes, $content ) {
 		$processor->set_attribute( 'data-id', $attributes['data-id'] );
 	}
 
+	$should_load_view_script = ( array_key_exists( 'behaviors', $attributes ) &&
```

Comment:
I believe we need to add a check not just at the block attribute level, but at the theme level.

Reply:
You're absolutely right! Sorry about this omission. I've updated the PR now with the logic that is replicated from [`block-supports/behaviors.php`](https://github.com/WordPress/gutenberg/blob/2c92b02fcf5d0911fc1355d910b276fde48195e2/lib/block-supports/behaviors.php#L48-L60).

Reply:
Ok great, tested and this seems to be working now 👍